### ISSUE TITLE
NCA | Handle Unlisted Non-Veteran Claimant

### ISSUE THREAD

**Issue by leikkisa** [department-of-veterans-affairs/caseflow#10906]
### Story
Approximately 98% of NCA appellants are not the Veteran as the Veteran has passed.  Users are being stopped from proceeding because they select the Veteran is not the claimant, but the claimant is not a dependent of the veteran.  The claimants are not verified claimants in VBMS.  Currently many NCA claims are being blocked because they can't select a claimant.

### Acceptance criteria
- [ ] Put this behind the feature toggle: [add_unlisted_claimant]
- [ ] If a user selects "National Cemetery Administration" as the benefit type, and they select that the claimant is someone other than the veteran, add a radio button option "Other", after any dependents are listed
- [ ] If the user selects "Other", show two text boxes below, one for "First name", and one for "Last name"
- [ ] If "Other" is selected, and First and Last name are populated, the user should be able to continue to the next step (validation that the claimant is populated should pass)
- [ ] The first and last names are saved in the database
- [ ] The current Claimant model delegation only applies if the claimant's decision review does not have the benefit type "nca".
- [ ] In Claimant, `bgs_record` and `bgs_address_service` should not call BGS unless participant_id is present.

### Technical notes
The `ClaimReviewIntake` model has code where these claimants are being created.  The `ClaimantValidator` has code for validating claimants.

* We may want to discuss whether to save these in the Claimant table, or a different table and what to call the fields.  That Claimant typically delegates these fields makes it more interesting.

If the engineer decides to save this into the same Claimant table, the "Not null" constraint on participant_id in the Claimant table needs to be removed.

## NCA POC 
Jennifer Kelly - NCA (New NCA Supervisor per Richard Byers II)
Email: Jennifer.kelly@va.gov

## Additional context/background
Source: Email from Nicholas VA - NCA - Caseflow & VBMS, originally from Richard Byers II
Note from Richard: We do have a workaround but it is tedious, time-consuming and not needed for PL 115-55 (i.e., it entails both NCA & VBA personnel, creating a VBMS claim, deleting it, etc., for almost every PL 115-55 case NCA has).

**Comment by laurjpeterson** [comment]
@leikkisa - circling back on this. what if i suggested options 1 and 2 you laid out to NCA to see if option 1 is viable for them?

**Comment by leikkisa** [comment]
@laurjpeterson I think we should suggest option 1 from the get go to unblock their work. They are tracking data on their own, so we can backfill the data for "veteran_is_not_claimant" later. Then we can code option 2, and backfill the data.  I do like eventually coding option 2, so that it's less confusing for users intaking the claims.

**Comment by msj0nes** [comment]
## Update 

New POC for NCA is Jennifer Kelly
Email:  Jennifer.kelly@va.gov

**Comment by leikkisa** [comment]
Moving potential solution notes into a comment (in order to replace them in the ticket description with the decided solution):

### Potential solutions
Have them select that the Veteran is the claimant
- This requires no code changes
- The person processing would need to look at the form to get the claimant's information
- This would make our data flawed, but whether that matters depends on how NCA is processing these and how the data is reported
- My understanding of this data is that it's primarily needed for creating EPs, so there may be no negative impact to the data being flawed in this way

Have them select the Veteran is not the claimant, but do not require them to select one of the Veteran's relationships
- Change code for claimant validation for NCA claims
- The person processing would need to look at the form to get the claimant's information
- We won't have any data about the claimant in Caseflow if for some reason we were to need it later, but we will have correct data that the Veteran is not the claimant

Have the user enter information about the claimant in Caseflow
- If the user selects "NCA", we would want to add an option for them to select information about the claimant. We'd want to know what fields they need (name? SSN? address?).  This is the biggest research/code lift, and gives us two paradigms for handling claimants, and could lead to inconsistencies with SHARE since we're not using the single source of truth for information about a person.
- This makes it possible to backfill data if eventually these claimants are added to SHARE
- We could make this information available to the person processing in Caseflow without looking at the form
